docs/upgrade.md

```diff
@@ -267,3 +295,57 @@ toc:
   - file: myfolder/page1
   - file: myfolder/page2
 ```
+
+## FAQ: Migration and Troubleshooting
+
+(known-limitations)=
+### What are the known limitations in Jupyter Book 2?
+
+Some features from Jupyter Book 1 are still in development for version 2:
+
+**Not yet available:**
+- **Editable Thebe cells** - Interactive code cells that users can edit and execute ([#443](https://github.com/jupyter-book/mystmd/issues/443))
+- **API documentation** - Sphinx autodoc-style API documentation generation ([#1259](https://github.com/jupyter-book/mystmd/issues/1259))
+- **Embed cell outputs as MyST** - Generate MyST from notebook outputs ([#2114](https://github.com/jupyter-book/mystmd/issues/2114))
```

Comment:
I'm not sure if it's also worth mentioning that the execution experience can also be improved upon. The experience is really at the boundary of questioning if it's worth switching over or still wait out:
https://github.com/jupyter-book/mystmd/issues/2019